### PR TITLE
Update preprint and README after acceptance

### PR DESCRIPTION
**Pull request by santisoler**
Update preprint disclaimer after acceptance. Update README.md after acceptance
to publication.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,24 +4,19 @@ by
...
 
 > Soler, S. R. and Uieda, L. (2021).
 > Gradient-boosted equivalent sources,
 > _Geophysical Journal International_.
-> doi: [xx.xxxx/xxxxxx](https://doi.org/xx.xxxx/xxxxxx)
+> doi: [10.1093/gji/ggab297](https://doi.org/10.1093/gji/ggab297)
 
-is available online at: [doi.org/xx.xxxx/xxxxxx](https://doi.org/xx.xxxx/xxxxxx)
--->
+is available online at: [doi.org/10.1093/gji/ggab297](https://doi.org/10.1093/gji/ggab297)
```
Comment (leouieda) [inline README.md:16]:
```suggestion
is available online at [doi.org/10.1093/gji/ggab297](https://doi.org/10.1093/gji/ggab297)
```

File: README.md [hunk 2]
```diff
@@ -4,24 +4,19 @@ by
...
 
-A preprint version is available on EarthArXiv at [doi.org/10.31223/X58G7C](https://doi.org/10.31223/X58G7C)
+A pre-copyedited, author-produced PDF of an article is available on EarthArXiv
```
Comment (leouieda) [inline README.md:18]:
```suggestion
An open-access, pre-copyedited, author-produced PDF of is available on EarthArXiv
```

**Review by leouieda — COMMENTED** [review]
Made some small tweaks to the disclaimer (added back the EarthArXiv link) and some suggestions for the README. Feel free to revert any changes and/or merge when ready 🙂

File: README.md (hunk 2 above)
Comment (santisoler) [reply README.md:18]: Open-access 💪🏼

**Comment by santisoler** [comment]
Great! I just moved the `\Year` variable to `variables.tex` and used it on the license notice instead of `\the\year`.

**Comment by santisoler** [comment]
I'm merging this!

**Pull request merged**